tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -37,55 +39,70 @@ bool IsRetriable(Status status) {
   }
 }
 
-Status CallWithRetries(const std::function<Status()>& f) {
-  int attempts = 0;
+Status CallWithRetries(const std::function<Status()>& f,
+                       const initial_delay_seconds) {
+  int attempt = 0;
   while (true) {
-    attempts++;
     auto status = f();
-    if (!IsRetriable(status) || attempts >= kMaxAttempts) {
+    if (!IsRetriable(status) || attempt > kMaxAttempts) {
```

Comment:
I think >= feels safer, even though in practice there should be no difference. So I'd prefer >=